server/src/routes/businessRoutes.ts

```diff
@@ -41,13 +42,25 @@ router.post('/', authMiddleware, async (req, res) => {
 router.get('/:id', async (req, res) => {
   try {
     const business = await Business.findById(req.params.id);
+
     if (business) {
       res.json(business);
     } else {
-      res.status(404).send('Business not found');
+      res.status(404).send('Business not found.');
     }
   } catch (err) {
-    res.status(500).json({ message: 'Error fetching business', error: err });
+    res.status(500).json({ message: 'Error fetching business.', error: err });
+  }
+});
+
+router.get('/search/:query', async (req, res) => {
```

Comment:
nono, ne :query, o query string params `https://stackabuse.com/get-query-strings-and-parameters-in-express-js/`